tests/daily_scraper/test_daily_scraper.py

```diff
@@ -29,6 +31,14 @@
     # -p win64 --branch=mozilla-central
     {'args': {'platform': 'win64',
               'branch': 'mozilla-central'},
+    'target': '2015-02-05-03-02-05-mozilla-central-firefox-38.0a1.en-US.win64.installer.exe',
+    'target_url': 'firefox/nightly/2015/02/2015-02-05-03-02-05-mozilla-central/firefox-38.0a1.en-US.win64.installer.exe'
```

Comment:
"# -p win64 --branch=mozilla-central" - this will take the latest right? So not sure if that folder will be latest.